test/other/metadce/test_metadce_files_wasmfs.sent

```diff
@@ -1,16 +1,16 @@
-a
-b
-c
-d
-e
-f
-g
-h
-i
-j
-k
-l
-m
-n
-o
-p
+/**@export*/a
```

Comment:
I wonder why these are minified but the other `.sent` files are not?

Reply:
The whitespace is not minified, but names are. This is `-O3 -g1` basically (and without closure).

Reply:
Oh I see its just the opt level.